### PR TITLE
Add k3s as an available option.  Change Rancher 2.0 to Rancher

### PR DESCRIPTION
**Pull request by JasonvanBrackel**
Added [k3s](https://k3s.io) as a viable Kubernetes option.  Removed 2.0 from the Rancher mentions as we're simply referring to it as Rancher.

### CODE REVIEW

**Comment by JasonvanBrackel** [comment]
Realized I did a couple commits from a clean ubuntu VM, they are mine.

**Comment by JasonvanBrackel** [comment]
I should be CLA signed I'm hoping it's flagging me simply because of the commit problem.

**Comment by DanyC97** [comment]
@JasonvanBrackel thank you for the PR, you mind fixing the conflict please?

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -69,7 +71,9 @@ cluster.
 
 * [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers) is a Terraform/Packer/BASH based Infrastructure as Code (IaC) scripts to create a seven node (1 Boot, 1 Master, 1 Management, 1 Proxy and 3 Workers) LXD cluster on  Linux Host.
 
-* [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) supports a nine-instance deployment on localhost with LXD containers.
+* [k3s](https://k3s.io) is a lightweight production-grade Kubernetes distribution.  With a super-simple installation process and a binary foot print around 40MB, it is ideal for local-machine development.
```
Comment (sftim) [inline content/en/docs/setup/pick-right-solution.md]:
```suggestion
* [k3s](https://k3s.io) is a lightweight production-grade Kubernetes distribution.  With a super-simple installation process and a binary footprint around 40MB, it is ideal for local-machine development.
```

**Comment by DanyC97** [comment]
thanks @sftim @JasonvanBrackel .
/lgtm

**Pull request merged**